frontend/app/forms/application-for-board-of-pardon-e/application-for-board-of-pardon-e.web.component.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Scoped } from "kremling";
+import TextInput from "../inputs/text-input.component.js";
+import FormThatPrints from "../inputs/form-that-prints.component.js";
+import Select from "../inputs/select.component.js";
+import Section from "../inputs/section.component.js";
+
+const paymentOptions = [
+  {
+    label: `Check, Money order, or Cashier's check  (Only to be filled out if application is mailed in. Check appropriate box for payment)`,
+    value: "check"
+  },
+  { label: "Credit card (domestic only)", value: "creditCard" }
+];
+
+const creditCardOptions = [
+  { label: "Visa", value: "visa" },
+  { label: "Mastercard", value: "mastercard" },
+  { label: "Discover", value: "discover" },
+  { label: "American Express", value: "amex" }
+];
+
+export default function ApplicationForBoardOfPardonExpungement_Web({ data }) {
+  return (
+    <Scoped css={css}>
```

Comment:
Same here about the Scoped component not supposed to have direct children that aren't dom elements. In this case you don't even have any css inside of the `css` string so it might be best to just remove this entirely.

```suggestion
```